tools/make-efi-testdata/README

```diff
@@ -0,0 +1,22 @@
+This directory contains a tool that is used to create the test
+data in the efi/testdata directory.
+
+It depends on a minimal set of source data (stored in the
+efi/testdata/src directory) that includes pre-generated RSA keys,
+public certificates, source code for mock EFI binaries and public
+dbx updates from uefi.org.
+
+It outputs event logs, EFI variable stores, EFI binaries and
+signature database updates for testing.
+
+Note that recreating the EFI binaries and signature databases
+is not entirely reproducible because their signatures include
+timestamps. This means they are recreated with different file
+hashes, but this doesn't affect the actual tests.
+
+Reproducibility of test data is dependent on the environment
+in which the data is built. This is particularly true for the
+EFI binaries, which are only reproducible when built with the
+same toolchain.
```

Comment:
I tried to rebuild the testdata and of course the tests then failed.

Reply:
Was it the tests in efi/bootmanager_policy_test.go that failed?

Reply:
yes, but secureboot_policy_test.go as well